spec/support/stubbed_constraint.rb

```diff
@@ -0,0 +1,9 @@
+RSpec.configure do |config|
+  config.before(:each, :with_find_constraint) do
+    allow_any_instance_of(FindConstraint).to receive(:matches?).and_return(true)
+  end
+
+  config.before(:each, :with_publish_constraint) do
+    allow_any_instance_of(PublishConstraint).to receive(:matches?).and_return(true)
+  end
```

Comment:
Can we brack these up to there own file and auto mark it

```
  config.define_derived_metadata(file_path: Regexp.new("/spec/features/publish")) do |metadata|
    metadata[:with_publish_constraint] = true
  end

  config.before(:each, :with_publish_constraint) do
    allow_any_instance_of(PublishConstraint).to receive(:matches?).and_return(true)
  end
```

?

Reply:
Nice idea. Done. I've removed the tag from `publish|find` feature directories